primitives/application-crypto/src/traits.rs

```diff
@@ -23,24 +23,22 @@ use sp_core::crypto::{CryptoType, CryptoTypeId, IsWrappedBy, KeyTypeId, Public};
 use sp_std::{fmt::Debug, vec::Vec};
 
 /// An application-specific key.
```

Comment:
Docs should be updated :P

Reply:
Missed that :-/
https://github.com/paritytech/substrate/pull/13709